README.md

```diff
@@ -291,6 +290,23 @@ It is up to the log operator to ensure that a satisfiable policy is configured,
 Synchronous Integration is provided by [`tessera.IntegrationAwaiter`](https://pkg.go.dev/github.com/transparency-dev/trillian-tessera#IntegrationAwaiter).
 This allows personality calls to `Add` to block until the new leaf is integrated into the tree.
 
+## Lifecyles
+
+### Appender
+
+This is the most common lifecycle mode. Appender allows the personality to add leaves, which will be sequenced
+contiguously with any prefix that the log has already committed to.
+
+This can be configured via [`tessera.NewAppender`](https://pkg.go.dev/github.com/transparency-dev/trillian-tessera@main#NewAppender).
+
+### Migration Target
+
+This mode is used when either:
+ - the log is being operated as a mirror of another log
```

Comment:
Currently it's only the 2nd of these - no checkpoint is ever written by the migration lib/tooling